apps/crn-server/test/fixtures/users.fixtures.ts

```diff
@@ -96,6 +96,7 @@ export const getGraphQLUser = (
   ...user,
   flatData: {
     alumniSinceDate: null,
+    alumniLocation: null,
```

Comment:
if there no tests for this in this PR and the only fixture change is the addition of a null `alumniLocation` to the fixture I think we need a test for when it comes back from squidex as not null, no?